charts/consul/test/unit/partition-name-configmap.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "partitionName/ConfigMap: disabled by default" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/partition-init-role.yaml  \
+      .
+}
+
+@test "partitionName/ConfigMap: enabled with global.adminPartitions.enabled=true and servers = false" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/partition-init-role.yaml  \
+      --set 'global.adminPartitions.enabled=true' \
+      --set 'server.enabled=false' \
+      . | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "partitionName/ConfigMap: disabled with global.adminPartitions.enabled=true and servers = true" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/partition-init-role.yaml  \
+      --set 'global.adminPartitions.enabled=true' \
+      --set 'server.enabled=true' \
+      .
+}
+
+@test "partitionName/ConfigMap: disabled with global.adminPartitions.enabled=true and global.enabled = true" {
```

Comment:
with `global.enabled = true` , servers should be enabled, and I *think* this test should instead enable the configMap?  Maybe I'm misreading?

Reply:
We dont enable the configmap in the server cluster, because we would fail if they changed the name to being something non-default.